dct_backend/dct.py

```diff
@@ -30,3 +34,54 @@ def save_food_item(self, name: str, station: str, meal_time: str,
         item = FoodItem(name, station, meal_time, calories, categories)
         self.menu.append(item)
         return item
+
+    """
+        incoming_dct_traffic adds people who are scanning into the dct
+
+        takeout: int
+            The current number of takeout scanning in
+
+        dinein: int
+            The current number of dinein scanning in
+
+        returns: Capacity
+            The total number of takeout, dineine, and total capacity
+            of takeout + dinein
+    """
+    def incoming_dct_traffic(self, takeout: int, dinein: int) -> Capacity:
+        self.takeout_capacity += takeout
```

Comment:
what is the "takeout_capacity" set to? or is it something that will be set in front end?

Reply:
what I imagined is ideally we would get this information from like a similar system that stores do it now where a store keeps track of the capacity by checking people in and keeping track of people leaving. so this would come from ideally from the computers the checkin people have at the dct and we would just need someone at the exit to keep track of people leaving